### PR TITLE
commit

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/test/java/com/nopcommerce/demo/cucumber/steps/MyStepdefs.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.nopcommerce.demo.cucumber.steps;
+
+import cucumber.api.java.en.Then;
+
+public class MyStepdefs {
+
+}
```
Comment (bhavika48) [inline src/test/java/com/nopcommerce/demo/cucumber/steps/MyStepdefs.java]: you can delete this file.not required

**Comment by bhavika48** [comment]
please update pr title

**Review by ghost — COMMENTED** [review]
I don't know how to update pr

File: src/test/java/com/nopcommerce/demo/cucumber/steps/MyStepdefs.java (hunk 1 above)
Comment (ghost) [reply src/test/java/com/nopcommerce/demo/cucumber/steps/MyStepdefs.java]: done

**Review by SaiLaxmiRaj — APPROVED** [review]

**Pull request merged**